src/features/advanced/cx-search-bar/cx-search-bar.ts

```diff
@@ -0,0 +1,22 @@
+import MaterialSearchAndResults from './MaterialSearchAndResults.vue';
+import $style from './cx-search-bar.module.css';
+
+function onTileReady(tile: PrunTile) {
+  subscribe($$(tile.anchor, C.ComExPanel.input), comExPanel => {
+    const actionBar = _$(comExPanel, C.ActionBar.container)!;
```

Comment:
```suggestion
  subscribe($$(tile.anchor, C.ComExPanel.input), async comExPanel => {
    const actionBar = await $(comExPanel, C.ActionBar.container);
```

Use `async/await $` where possible, as this function is type-safe and requires no `!` operator.